app/Http/Controllers/User/BookmarksController.php

```diff
@@ -78,8 +78,9 @@ public function index(Request $request, $user_id)
         $bible_id     = checkParam('bible_id');
         $book_id      = checkParam('book_id');
         $chapter      = checkParam('chapter|chapter_id');
-        $limit        = (int) (checkParam('limit') ?? 25);
-        $sort_by    = checkParam('sort_by');
+        // used by chapter annotations to get the max possible annotations for one chapter (180)
```

Comment:
comment doesn't  match code, and is different from chapter_max_verses. I would expect that each of the annotations would have a similar cap, and the code to enforce it would be in one place